### ISSUE TITLE
fund center graph design

### ISSUE THREAD

**Issue by stephnyc** [zoodigital/nia#31]
change the design of the fund center graph to be the same design as the one on the fund center

**Comment by seonzoo** [comment]
is this really necessary?

**Comment by stephnyc** [comment]
yes, according to the CEO (feel free to give an estimate of time, and we can discuss if its worth doing)

**Comment by stephnyc** [comment]
Seón - do you have a login for http://fundcentre.newireland.ie/?

**Comment by MartinZOO** [comment]
Martin to discuss further with George
